### PR TITLE
Hardening 03.10.2017

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: server/sonar-web/src/main/js/components/icons-components/EditIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <svg
+      className={className}
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 14 14"
+      width={size}
+      height={size}>
+      <path
+        fill="currentColor"
+        d="M3.918 11.918l0.711-0.711-1.836-1.836-0.711 0.711v0.836h1v1h0.836zM8.004 4.668q0-0.172-0.172-0.172-0.078 0-0.133 0.055l-4.234 4.234q-0.055 0.055-0.055 0.133 0 0.172 0.172 0.172 0.078 0 0.133-0.055l4.234-4.234q0.055-0.055 0.055-0.133zM7.582 3.168l3.25 3.25-6.5 6.5h-3.25v-3.25zM12.918 3.918q0 0.414-0.289 0.703l-1.297 1.297-3.25-3.25 1.297-1.289q0.281-0.297 0.703-0.297 0.414 0 0.711 0.297l1.836 1.828q0.289 0.305 0.289 0.711z"
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/components/icons-components/EditIcon.tsx]: MAJOR Exceeds maximum line length of 180 [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS103)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: server/sonar-web/src/main/js/apps/coding-rules/templates/rule/coding-rules-profile-activation.hbs [hunk 2]
```diff
@@ -7,7 +7,7 @@
     {{/if}}
   </div>
 
-  <div class="modal-body modal-container">
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/apps/coding-rules/templates/rule/coding-rules-profile-activation.hbs]: `.modal-container` is required to make sure the modal content fits on the screen. Try to add 10 form fields in this form, and you'll see the problem 😉

Comment (gregaubert) [reply server/sonar-web/src/main/js/apps/coding-rules/templates/rule/coding-rules-profile-activation.hbs]: Ok I didn't notice that we could have some parameters field, then the scrollbar should not be removed. I'll revert that change.

**Pull request merged**